magpie/__init__.py

```diff
@@ -1,273 +1 @@
-from __future__ import unicode_literals, print_function, division
```

Comment:
You know Python better than me, but as far as I've seen `__init__.py` files are almost always empty or just having imports while the real code is split into appropriately named files.

I don't care too much to be honest, this seemed a little cleaner to me :)